src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -59,6 +62,29 @@ private StringVerbatimFormatDefinition()
                 this.ForegroundColor = Colors.Maroon;
             }
         }
+
+#if dark_theme
+        private static readonly Color s_stringEscapeColor = Color.FromRgb(0xff, 0xd6, 0x8f);
+#else
+        private static readonly Color s_stringEscapeColor = Color.FromRgb(0x9e, 0x5b, 0x71);
+#endif
+
+        [Export(typeof(EditorFormatDefinition))]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.StringEscapeCharacter)]
+        [Name(ClassificationTypeNames.StringEscapeCharacter)]
+        [Order(After = ClassificationTypeNames.StringLiteral)]
+        [Order(After = ClassificationTypeNames.VerbatimStringLiteral)]
+        [UserVisible(true)]
+        [ExcludeFromCodeCoverage]
+        private class StringEscapeCharacterFormatDefinition : ClassificationFormatDefinition
+        {
+            private StringEscapeCharacterFormatDefinition()
+            {
+                this.DisplayName = EditorFeaturesResources.String_Escape_Character;
+                this.ForegroundColor = s_stringEscapeColor;
```

Comment:
>s_stringEscapeColor [](start = 39, length = 19)

How do themes work for other classifications in this file? 
For instance, I see that `StringVerbatimFormatDefinition` specifies `Colors.Maroon`. I assume there is a mechanism that overrides that for different themes. What is that mechanism?

Reply:
My preference is to remove the preprocessor stuff for dark theme here, and file an issue so we fix that other file promptly.

---
In reply to: [208754365](https://github.com/dotnet/roslyn/pull/28927#discussion_r208754365) [](ancestors = 208754365)

Reply:
they are two orthogonal issues to me.  We certainly need to file the followup issue.  However, some dev still needs to try out the actual colors and determine which work well . I did this with Kasey for the regex colors and you have to rebuild and retry things *constantly*.  Being able to quickly flip between things is super helpful.  I really do not want to remove the preprocessor stuff because that just means someone (or me) will just be adding it all back in later during that phase.

Reply:
How about leaving the preprocessor stuff for now, but include a note in the issue filed for themeing to remove it once done?

Reply:
I can do that :)